stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/adapter/viewholder/ChannelListItemViewHolderFactory.kt

```diff
@@ -1,58 +1,37 @@
 package io.getstream.chat.android.ui.channel.list.adapter.viewholder
 
 import android.view.ViewGroup
-import io.getstream.chat.android.ui.channel.list.ChannelListView
 import io.getstream.chat.android.ui.channel.list.ChannelListViewStyle
 import io.getstream.chat.android.ui.channel.list.adapter.ChannelItemType
 
 public open class ChannelListItemViewHolderFactory {
 
+    public lateinit var listenerContainer: ChannelListListenerContainer
+        internal set
```

Comment:
Ideally, these should be `protected` with `internal` setters, but unfortunately the setter can't be more public than the property itself.